skopt/gbt.py

```diff
@@ -50,4 +50,4 @@ def fit(self, X, y):
 
     def predict(self, X):
         """Predictions for each quantile."""
-        return np.vstack([rgr.predict(X) for rgr in self.regressors_])
+        return np.asarray([rgr.predict(X) for rgr in self.regressors_])
```

Comment:
What do you mean with data and argument dependent? It feels useful to be able to return as many/few quantiles as you'd want to instead of limiting yourself to mean and std. However there is no usecase for this yet.